completions_test.go

```diff
@@ -2466,6 +2466,7 @@ func TestCompleteCompletion(t *testing.T) {
 	expected := strings.Join([]string{
 		"bash",
 		"fish",
+    "nushell",
```

Comment:
The indentation is wrong here, which makes the linter fail

Reply:
Ugh... neovim shows the spacing correct, but yes different whitespace. fixed.